draft-meynell-panrg-scion-research-questions.md

```diff
@@ -201,11 +201,36 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 * One AS belongs to many ISDs?
 
 
+## Beacon Forwarding Policy
+
+The idea behind "beaconing" is to discover all possible paths (loop free and with a fixed maximum length) between two CORE AS.
+Every AS forwards any received beacons to all neighbor CORE ASes unless this would cause a loop or exceed the fixed maximum length.
+Implemented naively, the number of paths (and beacons) grows exponentially with the network size.
+This is currently mitigated, primarily (and efficiently), by forwarding only the five "best PCBs" to neighbours.
+
+This "best PCB" practice has several implications:
+
+* It limits the number of available paths for endpoints such that only five truly diverse paths may be available to a given destination.
+  "truly diverse" means: Most ASes will receive five PCBs to a given destination from several of their neighbours. However, assuming
+  those ASes are use a similar PCB selection policy, it is not unlikely that each neighbor AS will forward a very
+  similar set of five PCB. In effect, there may be more than five paths available, but they are likely to diverge only on the first
```

Comment:
For this to happen, these neighbors have to share a common trailing path, e.g., being X our AS:
```
1) a->b->c -> X
2) a'->b'->c' -> X
etc.
```
Paths 1 and 2 would be "similar" if a' ≈ a and b'≈ b, while having c and c' being different neighbors.
Definitely not an impossible situation, but also not common: most likely the trailing path of neighbor c will be different that that of neighbor c', either in AS numbers or interface IDs.

Reply:
Just to clarify the example, we are looking for paths from X to Y:
```
1) Y->a->b->c -> X
2) Y->a'->b'->c' -> X
```
At a first step I'd argue that it is quite likely that, going from Y->X,  we have a'=a. If a=a' then the same argument applies to b=b' and so forth.
Consequently I'd argue that there is a high probability of overlap.

Unfortunately, I don't think there is any analysis or experiments to confirm/reject this. If people think this is wrong then we can remove it. Or I can try to rephrase it as a research question that would need to be clarified whether this may be a problem or not?